File: 1020.c
/*  
    @autor: Guiventura6;
    @data: 21/01/2022;
    @Problema: Age in Days;
*/

#include <stdio.h>

int main(){
    
    int ageInDays, years, months, days, rest;

    scanf("%d", &ageInDays);

    years = ageInDays / 365;    
    rest = ageInDays % 365;    
    months = rest / 30;    
    days = rest % 30;

    printf("%d ano(s)\n", years);    
    printf("%d mes(es)\n", months);
    printf("%d dia(s)\n", days);
    
    return 0;
}

File: 1020.py
''' 
    Autor: Guiventura6;
    Data: 03/03/2022;
    Problema: Age in Days;
    Resume: Program that reads age in days and convert to years, months and days.
    obs: The whole year with 365 days and 30 days every month.
''' 

def convert_age(age):
    years = age // 365
    months = (age % 365) // 30
    days = (age % 365) % 30
    return years, months, days
    
ageInDays = int(input())
years, months, days = convert_age(ageInDays)

print('{} ano(s)\n{} mes(es)\n{} dia(s)'.format(years, months, days))

